backend/data/road_segmenting/nvdbapi-clientinfo.json

```diff
@@ -0,0 +1,2 @@
+{ "X-Client" : "YOUR SYSTEM",
```

Comment:
Change this to be a .gitignored file, and make an example file of it like "local-example.py" for settings. Add in to README that this is a file that needs to be filled out, because yada-yada. 
I get that we don't need this right now during development, but we're committing code that will maybe be in prod in the future, and we should configure with that in mind.